### DISCUSSION TITLE
Other events of interest

### DISCUSSION THREAD

**Discussion by pburkholder** [GSA-TTS/federal-platform-engineering-cop#11] [General]
I saw an announcement for [ATARC's DevSecOps Summit](https://atarc.org/event/open-source-summit/) coming up on 6/13, and thought maybe we should start a thread for events of related interest.

**Comment by brianrandallfox** [comment]
Well, there's the DevOps Days in DC coming up!   https://devopsdays.org/events/2024-washington-dc/welcome/
